lms/static/sass/base/_base.scss

```diff
@@ -130,7 +130,6 @@ a:focus {
   margin: 0 auto 0;
   padding: ($baseline*2) 0;
   max-width: grid-width(12);
-  min-width: 760px;
```

Comment:
@talbs,
would it be better if this was included in `/lms/static/sass/shared/_course_object.scss` since I'll be defining style rules that will be included in homepage, find courses page and course results page (last two will probably be the same page)?
`/lms/static/sass/shared/_course_object.scss` already defines style rules for every element that is nested within the section. I think that it would be easier to maintain it if we only had one file that defines style for the section which has course cards nested within. Any thoughts or suggestions?